doc/vault-integration.md

```diff
@@ -43,29 +43,33 @@ Commands below assumes the VM is an Ubuntu instance.
 
 #### Install Vault
 Run the following commands in your VM.
+
+Install vault on Ubuntu following the vault [docs](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install)
+
+Configure vault
 ```bash
-# install vault and enable the service
-sudo apt update && sudo apt install vault
 sudo systemctl enable vault.service
 
 # create a configuration file for vault. NOTE: this creates a vault service with TLS disabled. 
 # This is done to make the configuration step easy to follow only. TLS should be enabled for real workloads.
-cat <<< "ui = true
-
+cat <<< 'ui = true
+disable_mlock = true
 storage "file" {
   path = "/opt/vault/data"
 }
 
 listener "tcp" {
   address = "0.0.0.0:8200"
   tls_disable = 1
-}" | sudo -u vault tee /etc/vault.d/vault.hcl > /dev/null
+}' | sudo -u vault tee /etc/vault.d/vault.hcl > /dev/null
 
 sudo systemctl start vault.service
 
 export VAULT_ADDR='http://127.0.0.1:8200'
 # This command will print out unseal keys and the root token.
 vault operator init
+# record the token
+export VAULT_TOKEN="<REPLACE>"
```

Comment:
not sure about this bit here. probably better to separate it out.

without having to export the token, users can just copy and paste the entire script into the terminal and run it. but now, they will need to do it step by step which could be confusing.

Reply:
@nimakaviani The user can't copy and paste the entire script as is today, when they get to the following lines calling `vault` like `vault operator unseal` it fail.
I can remove this line and leave it as is.